README.md

```diff
@@ -283,3 +283,94 @@ Not all required parameters are provided, since deploy specific, see [report-gen
+-   if one wants to model that another Mandataris replaces the first, another Mandataris entity `R` is created and the new 'verhinderd' entity `V` receives a `mandaat:isTijdelijkVervangenDoor`  reference to `R`.
+
+This means that except for correcting mistakes and changing the `mandaat:einde`, Mandataris entities should never be modified.
+
+## Expected Peripheral Knowledge
+
+There is a substantial amount of data that is referenced in the entities on the LDES above but that is not expected to change often (or at all) and that isn't the responsibility of the LMB system to maintain. Let's call such data Peripheral Knowledge.
+
+As the LMB LDES instances reference this data, we do expect systems that the LDES data to have this Peripheral Knowledge loaded themselves. 
+
+A suggestion would be that a central system could provide its own LDES feed that other systems can use to update their peripheral knowledge (given how little this data is expected to change, this can even be a manual update). There are multiple options here and this is something that can wait as we will likely only launch centrally first (possibly one separately with heavy, heavy support). In this case, a migration or set of migrations will be good enough. Another path is that we can make an interface (API, not LDES) where they just fetch the latest version. Another path is to use the caching headers of LDES, allowing caching for e.g. half a year. We can also terminate an LDES when we know no more data is going to change.
+
+## LDES setup
+
+The LDES spec can be found [here](https://semiceu.github.io/LinkedDataEventStreams/). 
+We will be using the implementation provided by [redpencil.io on their github](https://github.com/redpencilio/fragmentation-producer-service) to publish the LDES. It will be fed using our own service that monitors deltas on our SEAS instance.
+
+We'll use a time-fragmenter, one stream will be set up per type of instance that we will share, with pagination and the following setup for versioning: 
+
+    <stream> ldes:timestampPath prov:generatedAtTime ;
+             ldes:versionOfPath dct:isVersionOf .
+             ldes:retentionPolicy ext:retention .
+    
+    ext:retention a ldes:DurationAgoPolicy ;
+          tree:value "P1Y"^^xsd:duration .
+
+This also means that we promise that we will keep the data for at least one year after it was created.
+The url of the type of entity will be based on the resources path of the entity type in 
+
+## Links to Entities
+
+When other entities are referenced from an LDES feed instance, the original URI of that entity is used, not the versioned URI.
+
+For instance, if this snippet is part of a `mandaat:Mandatarais` instance that refers to a `person:Person`, its `mandaat:isBestuurlijkeAliasVan` predicate refers to the true URI of the person, not the person's versioned URI (if any exists because we had to create it ourselves).
```

Comment:
I'm a bit confused about the versioned uri's of persons. Above I seem to only find info about versioned mandataris?